File: 485. Max Consecutive Ones/485. Max Consecutive Ones.java
public class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int cnt = 0, maxcnt = 0, i = 0, l = nums.length;
        for (i = 0; i < l; i++){
            if (nums[i] == 1)
                cnt++;
            else{
                maxcnt = (maxcnt > cnt ? maxcnt : cnt);
                cnt = 0;
            }
        }
        return maxcnt > cnt ? maxcnt : cnt;
    }
}

File: 485. Max Consecutive Ones/485. Max Consecutive Ones.swift
class Solution {
    func findMaxConsecutiveOnes(_ nums: [Int]) -> Int {
        var lengthofArray = nums.count
        var cnt = 0, maxcnt = 0, i = 0
        while i < lengthofArray {
            if nums[i] == 1 {
                cnt += 1
            }
            else {
                maxcnt = maxcnt > cnt ? maxcnt : cnt
                cnt = 0
            }
            i += 1
        }
        return maxcnt > cnt ? maxcnt : cnt
    }
}
